vscode/src/client/analyzerClient.ts

```diff
@@ -186,6 +228,10 @@ export class AnalyzerClient {
     );
   }
 
+  public isServerRunning(): boolean {
```

Comment:
I think I simply moved this function to a place that made more sense.  Plus it doesn't appear to be used anywhere, so maybe it could just be removed? 

I just wonder if it is worth keeping just in case the state gets out of sync because I manually `kill -9` the kai rpc server process (probably out of spite).